_layouts/indicator.html

```diff
@@ -45,17 +45,19 @@ <h2>{{ meta.graph_title | t }}</h2>
   data-charttitle="{{ meta.graph_title | t }}" data-measurementunit="{{ meta.computation_units | t }}" data-datasource="{{ meta.source_organisation_1 | t }}" data-geographicalarea="{{ meta.national_geographical_coverage | t }}" data-footnote="{{ meta.data_footnote | t }}" data-showdata="{{ show_data }}" data-graphtype="{{ meta.graph_type }}" data-geocoderegex="{{ meta.data_geocode_regex }}" data-showmap="{{ meta.data_show_map }}">
     {% if show_data %}
     <div class="col-md-3">
-      <div id="toolbar">
-        <span id="series-help">
-          <h4>{{ t.indicator.sub_categories }}</h4>
-          <p>{{ t.indicator.series_help }}</p>
-        </span>
-        <p>{{ t.indicator.legend_help }}</p>
-        <div id="fields">
-          <p class="async-loading" data-img="{{ site.baseurl }}/assets/img/loading-small.gif"></p>
+      <div id="chartControls" show="1">
```

Comment:
I'm not a huge HTML-validation nitpicker, but just to be thorough I think this should be `data-show` instead of `show`, right?